### ISSUE TITLE
mksrf_landuse_histclm50_LUH2_2000.c170412.nc missing in inputdata repository

### ISSUE THREAD

**Issue by olyson** [ESCOMP/CTSM#1894]
### Brief summary of bug

The file mksrf_landuse_histclm50_LUH2_2000.c170412.nc is missing in the svn inputdata repository.  Several users are getting errors related to this when creating surface datasets:

(https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/rawdata/pftcftlandusedynharv.0.25x0.25.MODIS.simyr1850-2015.c170412/)
### General bug information

**CTSM version you are using:** NA

**Does this bug cause significantly incorrect results in the model's science?** No

**Configurations affected:** NA

### Details of bug

It looks like most of the files that are here below are missing in the repository:

/glade/p/cesmdata/cseg/inputdata/lnd/clm2/rawdata/pftcftlandusedynharv.0.25x0.25.MODIS.simyr1850-2015.c170412

**Comment by slevis-lmwg** [comment]
I came across this issue, so I submitted:
`./rimport -file lnd/clm2/rawdata/pftcftlandusedynharv.0.25x0.25.MODIS.simyr1850-2015.c170412/mksrf_landuse_histclm50_LUH2_2000.c170412.nc`
to add this file to the repo.

Actually... I decided to add all the mksrf_landuse files from this directory to the repo, so that we may close the issue. The process will take the next hour or two to complete.

@olyson this directory also contains
- three mksrf_lai files
- four mksrf_soilcolor files

Should I add any of them to the repo?
